### ISSUE TITLE
Website that does reflow text, but not properly.

### ISSUE THREAD

**Issue by Grossdm** [siyangqiu/FFTextWrap#2]
I happened upon a website that was in need of having its text reflowed by your addon:
http://www.faqs.org/faqs/compression-faq/part1/section-8.html

The text did reflow, but not properly. It appeared that your addon incorrectly thought that the screen (or viewport - I'm cool because I know terms like that) was wider than it actually was.

Please look into this website and see if you a reproduce the behavior.

Thank you,
Doug®

**Comment by siyangqiu** [comment]
Thanks for letting me know! 

So here's what's actually happening on the website: the text that is not reflowing is held within \<pre> tags. This tag is used for text that is meant to be pre-formatted - it keeps the line spacings and line breaks of the text. This also means that if we specify a maximum width for the container, the text would simply overflow. Check out this link for more [details.](https://www.sitepoint.com/everything-need-know-html-pre-element/)

In short, reflow is working as intended, but it isn't reflowing text on the page because this page is using \<pre>. 

Let me know if you have any suggestions on how you want the extension to handle \<pre> tags.

**Comment by Grossdm** [comment]
Oh, `<pre>` tags/blocks.
I've messed with them a bit with user stylesheets in the past.

Wrapping might make sense on a monitor, especially if you could have the wrapped text with a hanging indent.

For my phone, I think I'd prefer scrolling.
If it was something I wanted to really study, I'd turn my phone to landscape for the best representation.


It doesn't seem right for a `<pre>` to dictate the width of the entire page, though.

I hope those thoughts are helpful.

**Comment by Grossdm** [comment]
I had to edit that, because my GitHub was interpreting the text as html! :grin:
